response_operations_social_ui/templates/partials/footer.html

```diff
@@ -0,0 +1 @@
+<footer class="footer"></footer>
```

Comment:
Do we need this?

Reply:
The intention is to add the internal service footer at some point.  I think even a blank footer is useful to reassure users they've hit the end of the page.